docs/source/en/add_new_pipeline.mdx

```diff
@@ -9,7 +9,7 @@ Unless required by applicable law or agreed to in writing, software distributed
 an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 -->
 
-# How to add a pipeline to 🤗 Transformers?
+# How to create a custom pipeline?
```

Comment:
Is there a reason you're replacing that part instead of adding a new part ?

I think that documentation part is important and has merit as it also partly explains how current pipelines work, how they were designed.

We could for sure rephrase things since not a lot of community members have actually added any pipeline, but the content describing the pipeline's structure (without talking about custom ones) shouldn't be lost IMO.

Reply:
The content on how to add it to Transformers is still there, but the page has evolved to show how to create a custom pipeline and share it without necessarily adding it to Transformers. I can add a summary where this still said at the top of the file however.

Reply:
diff got me on this ! You're right. It's fine then.